src/clr-angular/popover/dropdown/dropdown-item.ts

```diff
@@ -31,7 +31,7 @@ export class ClrDropdownItem implements AfterViewInit {
 
   private unlisten;
 
-  @Input()
+  @Input('clrDisabled')
```

Comment:
turns out we can't use the `disabled` attribute anymore for dropdown item since IE won't allow focus on any element that has this. I'm renaming to `clrDisabled` here so it'll still auto attach the right class name and `aria-disabled` value.